### ISSUE TITLE
.sam + .pps?

### ISSUE THREAD

**Issue by taptroupe** [Nisto/musyx-extract#3]
Hello, I stumbled across your MusyX tool and I'd like to thank you for making it. I'm pretty new to extracting files, and these game-exclusive filetypes are driving me mad.

I'm trying to extract sound effects from _Lost Kingdoms_, and the SFX seems to be a duo of .sam and .pps files, instead of .sam and .sdi. Due to that, it doesn't seem to work with your tool. I was wondering if there's any way to work with these .pps files, so I've attached a link to some of the files below. I appreciate your reading this, and hope to hear a response from you. 

Thank you very much. (and stay safe!)
[sound sample.zip](https://github.com/Nisto/musyx-extract/files/4534584/sound.sample.zip)

**Comment by Nisto** [comment]
@taptroupe Hi. Sorry for the delayed response. GitHub is not great at notifying when things like new Issues are created it seems... Anyway, I have just uploaded a [script](https://github.com/Nisto/extractors/blob/master/lost_kingdoms_pps.py) here that will extract these PPS files. Please mention me (@Nisto) if this helps so I can close this Issue.

**Comment by taptroupe** [comment]
It works perfectly, thank you very much! 

I've been able to extract the .pps files with your script, and it works without a hitch with your MusyX extractor. It's very, very much appreciated for your help in this issue - feel free to close it @Nisto .

I have attached an image of Kirby to express my gratitude.
![D_zra3vVAAA7Cq6](https://user-images.githubusercontent.com/64342555/82300984-7532c280-996c-11ea-9c7d-97fd6c843426.png)
